lib/config/error-codes/api.json

```diff
@@ -130,6 +145,181 @@
           "class": "BadRequestError"
         }
       }
+    },
+    "bulk": {
+      "code": 7,
+      "errors": {
+        "document_creations_failed": {
+          "code": 1,
+          "message": "Some document creations failed: %s",
+          "class": "PartialError"
+        },
+        "data_not_imported": {
+          "code": 2,
+          "message": "Some data was not imported: %s",
+          "class": "PartialError"
+        }
+      }
+    },
+    "security": {
+      "code": 8,
+      "errors": {
+        "role_not_found": {
+          "code": 1,
+          "message": "Role with id %s not found",
+          "class": "NotFoundError"
+        },
+        "profile_not_found": {
+          "code": 2,
+          "message": "Profile with id %s not found",
+          "class": "NotFoundError"
+        },
+        "user_not_found": {
+          "code": 3,
+          "message": "User with id %s not found",
+          "class": "NotFoundError"
+        },
+        "admin_exists": {
+          "code": 4,
+          "message": "Admin user is already set",
+          "class": "KuzzleError"
```

Comment:
should be a PreconditionError (the error currently thrown by Kuzzle is wrong, this is not an internal problem)